src/index.html

```diff
@@ -1,13 +1,373 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#menu" class="icon icon--menu header__menu-opener"></a>
+            <a href="/" class="logo">
+              <img
+                src="./images/logo.png"
+                alt="Condo Kings"
+                class="logo__image"
+              >
+            </a>
+            <a href="tel:+1 234 555-55-55" class="icon icon--phone header__call"></a>
+          </div>
+          <h1 class="header__title">
+            Miami
+            <br>
+            Condo Kings
+          </h1>
+        </div>
+      </div>
+    </header>
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="#" class="icon icon--cross header__menu-opener"></a>
+            <a href="/" class="logo">
+              <img
+                src="./images/logo.png"
+                alt="Condo Kings"
```

Comment:
Краще повну назву написати, я думаю
```suggestion
                alt=" Miami Condo Kings"
```